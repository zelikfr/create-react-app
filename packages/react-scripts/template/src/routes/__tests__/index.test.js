import React from 'react'
import { shallow } from 'enzyme'
import Routes from '../'

describe('Routes', () => {
  it('should create a component wrapped with Routes', () => {
    // When I render the Routes element
    const component = shallow(<Routes />)
    // Then I should have the component to match the snapshot.
    expect(component).toMatchSnapshot()
  })
})
