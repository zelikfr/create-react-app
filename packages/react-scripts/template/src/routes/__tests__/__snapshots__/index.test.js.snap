// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes should create a component wrapped with Routes 1`] = `
<BrowserRouter>
  <AuthProviderComponent>
    <Switch>
      <Route
        component={[Function]}
        exact={true}
        path="/"
      />
      <Route
        component={[Function]}
        path="/login"
      />
    </Switch>
  </AuthProviderComponent>
</BrowserRouter>
`;
